learn/git-training-01/index.qmd

```diff
@@ -0,0 +1,102 @@
+|-----------:|:---------------------------------------------------------|
+| 2:00 - 2:05 | Introduction                                                               |
+|       15min | What is version control and why should I use it for Open Science projects? |
+|        5min | The general workflow                                                       |
+|       10min | How do I set up to start using Git?                                        |
+|       10min | Where does Git store information?                                          |
+|       30min | How do I record changes in Git?                                            |
+|        5min | _Break away from the screen_                                                 |
+|       20min | How do I record notes about my changes and why?                   |
+|       30min | How can I identify old versions of files? How do I review my changes?      |
+|       10min | How can I tell Git to ignore files I don't want to track?                  |
+|        5min | _Break away from the screen_                                                 |
+|       20min | How do I share my changes with others on the web?                          |
+|       30min | How can I use version control to collaborate with others?            |
+|       15min | What do I do when my changes conflict with someone else's?                 |
+| 5:30 - 5:35 | Closing                                                                    |
+
+# Materials you will need
+
+We will use Git within the Rstudio graphical interface and GitHub. For this you will need to use these instructions:
+
+-   Install R and Rstudio <https://posit.co/download/rstudio-desktop/>
+-   Install Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>
+-   Create a GitHub account <https://github.com/signup>
+
+# Materials we used
+
+- [Slides that we used in the training](https://docs.google.com/presentation/d/1kE13oZCoSwlXJL_UhvhiX6W0hNaE1ZEkFsHAX-Pl5o0/edit?usp=sharing) with notes expanding the content for instructors
+- [Template for our document with shared notes](https://docs.google.com/document/d/1p7BWfqG0-advDHmhzYas1WyhHIZQvLs1HWR39_o7X7k/edit?usp=sharing)
+- [Template for our pre-training form](https://docs.google.com/forms/d/1LsSsMmwuLIr_GCsw1K5AUxfCMb5_XiAlpno7up6zc8Q/copy)
+- [Template for our feedback form at the end of the training](https://docs.google.com/forms/d/1h-mHJgPg0u7Www6D2-kz4Q3scQ03M30IYLm-0d6QIv8/copy) 
+- [Video of the last version of the training](https://drive.google.com/file/d/1yAztO0qb13Jo6qwQ-PN7tHgTbGcKv8LX/view?usp=share_link)
+
+# Any questions?
+
+If you need any assistance installing the software or have any other questions about the training, please send an email to [andree.valle-campos\@lshtm.ac.uk](mailto:andree.valle-campos@lshtm.ac.uk){.email}
+
+# Attribution
+
+This page format is inspired by [Metadocencia](https://www.metadocencia.org/curso/intro-abc-online/)
```

Comment:
Possibly also credit the illustration picture's author here

Reply:
Thank you, Hugo, for your suggestion! I recently saw it for episoup too. Fixed in https://github.com/epiverse-trace/epiverse-trace.github.io/pull/61/commits/fdd6a579fd42ff64e1847c5e48aaddbbf2d74ae7